### PR TITLE
Add uac qid service

### PR DESCRIPTION
**Pull request by NickGrantX**
# Motivation and Context

We have replaced the IAC service with a new UAC QID service. We need to deploy this to our CI/CD environments automatically, as well as building the docker image on our own infrastructure.

# What has changed

Swapped out IAC for UAC QID service.

# How to test?

Fly the pipelines. Run the pipelines.

# Links

Trello: https://trello.com/c/8glYoovR

# Screenshots (if appropriate):

### CODE REVIEW

**Comment by NickGrantX** [comment]
[Deploy new UAC/QID service (8)](https://trello.com/c/8glYoovR/901-deploy-new-uac-qid-service-8)

File: pipelines/ci-sit-kubernetes-pipeline.yml [hunk 1]
```diff
@@ -88,10 +88,10 @@ resources:
     username: _json_key
     password: ((gcp.service_account_json))
 
-- name: iacsvc-docker-latest
+- name: uac-qid-service-docker-latest
   type: docker-image
   source:
-    repository: eu.gcr.io/census-rm-ci/rm/census-rm-iacsvc
+    repository: eu.gcr.io/census-rm-ci/rm/census-rm-uac-qid-service
```
Comment (jcox-dev) [inline pipelines/ci-sit-kubernetes-pipeline.yml]:
```suggestion
    repository: eu.gcr.io/census-gcr-rm/rm/census-rm-uac-qid-service
```

File: pipelines/docker-build-pipeline.yml [hunk 2]
```diff
@@ -64,21 +64,19 @@ resources:
...
 
-  # TODO: requires new iac-service repo to start releases
-  # - name: iac-service-release
-  #   type: github-release-latest
-  #   source:
-  #     owner: ONSdigital
-  #     repository: iac-service
-  #     access_token: ((github.access_token))
-  #     order_by: time
+ - name: uac-qid-service-release
```
Comment (jcox-dev) [inline pipelines/docker-build-pipeline.yml]: I think this whole release block needs moving in(?) one space to align with the others

**Review by AdamHawtin — APPROVED** [review]
:+1: Apart from issues Jamie has spotted LGTM

**Review by jcox-dev — CHANGES_REQUESTED** [review]
Couple of tweaks but looks pretty much ready to fly

File: pipelines/docker-build-pipeline.yml (hunk 2 above)
Comment (jcox-dev) [inline pipelines/docker-build-pipeline.yml]:
```suggestion
  - name: uac-qid-service-release
```

**Review by jcox-dev — CHANGES_REQUESTED** [review]

File: pipelines/docker-build-pipeline.yml [hunk 3]
```diff
@@ -643,19 +641,19 @@ jobs:
       get_params:
         skip_download: true
 
-  - name: build-iac-service-rm-census
+  - name: build-uac-qid-service-master
     plan:
-    - get: iac-service-rm-census
+    - get: uac-qid-service-rm-census
```
Comment (jcox-dev) [inline pipelines/docker-build-pipeline.yml]:
```suggestion
    - get: uac-qid-service-master
```

**Review by jcox-dev — CHANGES_REQUESTED** [review]
Resource name mis-match

**Review by jcox-dev — APPROVED** [review]

**Pull request merged**